### DISCUSSION TITLE
Welcome to NubesGen

### DISCUSSION THREAD

**Discussion by cmaneu** [microsoft/NubesGen#42] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LayZeeDK** [comment]
Hello, I'm a Microsoft MVP in Developer Technologies from Denmark.

I'm mostly interested in JavaScript development but I also use Azure and .NET at work.

**Reply by jdubois** [reply]
Welcome :-)

**Reply by cmaneu** [reply]
Welcome @LayZeeDK :)

**Reply by sadranyi** [reply]
welcome
